### PR TITLE
feature: translate trix field controller js warnings

### LINKED ISSUE #3329: Locale / translate Trix js messages
### Feature
When trying to upload an image in a not saved resource, I see a js alert message:

Image

It should be possible to change or add a translation to the message.

### Current workarounds
- Create a new trix_field_controller.js with the translated message?
- Replace event trix-file-accept?

### Screenshots or screen recordings
Sent one above

### Additional context
I can implement if there is a direction on how or where to set the message to translate.

### PR DESCRIPTION
**Pull request by rafaeliga**
# Description
Translates trix_field_controller js warnings

Fixes #3329

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [documentation](https://github.com/avo-hq/avodocs)
- [x] I have added tests that prove my fix is effective or that my feature works

## Screenshots & recording
image

## Manual review steps

1. Change the locale to one of locales supported
2. Try to upload a image in trix when creating a new resource

Manual reviewer: please leave a comment with output from the test if that's the case.

## Documentation

https://github.com/avo-hq/docs.avohq.io/pull/301

### CODE REVIEW

**Comment by rafaeliga** [comment]
@Paul-Bob Your steps to implement the feature worked great!

I added the translation for the others 2 warnings that I found there.

Can you help with the tests if necessary for this change?

**Comment by Paul-Bob** [comment]
Hey @rafaeliga thank you for the contribution, is looking great!

We already have some tests for the trix field on `spec/system/avo/trix_field_spec.rb`

That test is already a system test and you should be able to test this alerts inside it.

**Comment by rafaeliga** [comment]
@Paul-Bob Do you use VS code?

I see that the project uses Ruby LSP, but I think its missing something to configure here:

image

**Comment by Paul-Bob** [comment]
> @Paul-Bob Do you use VS code?
> 
> I see that the project uses Ruby LSP, but I think its missing something to configure here:

Yes, I'm not sure why that is happening on your side.

image

File: spec/system/avo/trix_field_spec.rb [hunk 1]
```diff
@@ -35,6 +35,13 @@
 
         expect(find_field_value_element("body")).to have_text "Works for us!!!"
       end
+
+      it "contains js alert messages translated" do
```
Comment (rafaeliga) [inline spec/system/avo/trix_field_spec.rb:39]: @Paul-Bob it seems the data attribute it not showing in the test...do you know why?

Comment (Paul-Bob) [reply spec/system/avo/trix_field_spec.rb:39]: I'm not sure why, @rafaeliga. Have you checked the DOM locally? Is it rendered on DOM, like you're trying to check it on the test?

Comment (rafaeliga) [reply spec/system/avo/trix_field_spec.rb:39]: Oops, you are right, I was checking the wrong data key. Thanks!

**Comment by rafaeliga** [comment]
@Paul-Bob Which page should I add documentation?

Maybe this one? https://docs.avohq.io/3.0/i18n.html#customize-the-locale

**Comment by Paul-Bob** [comment]
> @Paul-Bob Which page should I add documentation?
> 
> Maybe this one? https://docs.avohq.io/3.0/i18n.html#customize-the-locale

Let's add a new section on the Trix field docs (https://docs.avohq.io/3.0/fields/trix.html) and specify the supported translation keys and when those are used.

**Comment by rafaeliga** [comment]
@Paul-Bob Looks good?

**Review by Paul-Bob — APPROVED** [review]
Looks great, thank you for this contribution @rafaeliga!

**Pull request merged**